lib/rubygems/gemcutter_utilities.rb

```diff
@@ -285,15 +302,8 @@ def get_mfa_params(email, password)
     params
   end
 
-  def get_user_mfa_level(email, password)
-    response = rubygems_api_request(:get, "api/v1/profile/me.yaml") do |request|
-      request.basic_auth email, password
-    end
-
-    with_response response do |resp|
-      body = Gem::SafeYAML.load clean_text(resp.body)
-      body["mfa"]
-    end
+  def get_user_warning(profile)
+    profile["warning"]
```

Comment:
This method seems unused? It should be either removed or the code changed to use it.